### PR TITLE
User APIs documentation for 3.9.0

### PR DESCRIPTION
**Pull request by sreeragksgh**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sreeragksgh,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://755.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://755.qa.docs.sunbird.org/3.9.0/apis/userapi// <br />  <br/> **Modified sections are -** <br />

File: apis/userapi/userapi.yaml [hunk 1]
```diff
@@ -1937,78 +1950,234 @@ paths:
...
-                            isApproved:
-                            orgLeftDate:
-                        provider:
-                        countryCode:
-                        maskedEmail: se*************@yopmail.com
-                        email: se*************@yopmail.com
+                            isRejected: null
+                            locations:
+                              - code: 29
+                                name: Karnataka
```
Comment (Sunitashelar) [inline apis/userapi/userapi.yaml:2006]: @sreeragksgh @mohitga request that we do not use actual names, etc. in Sunbird documentation. Can we replace 'Karnataka' 'Belgavi' etc. with variable names providing these as examples? This is a global comment. Further down there are references to other districts, States and towns too.

File: apis/userapi/userapi.yaml [hunk 2]
```diff
@@ -1937,78 +1950,234 @@ paths:
...
+                        countryCode: null
+                        tncLatestVersionUrl: https://sunbirdstagingpublic.blob.core.windows.net/termsandcondtions/terms-and-conditions-v9.html#termsOfUse
+                        maskedEmail: k1**@yopmail.com
+                        email: k1**@yopmail.com
+                        rootOrg:
+                          dateTime: null
+                          preferredLanguage: null
+                          keys:
+                          approvedBy: null
+                          channel: dikshapreprodcustodian
```
Comment (Sunitashelar) [inline apis/userapi/userapi.yaml:2086]: @sreeragksgh @mohitga please remove all references to diksha and any specific servers unless necessary from Sunbird documentation.

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

**Pull request merged**